proto/src/determined/api/v1/experiment.proto

```diff
@@ -352,6 +352,8 @@ message TrialsSnapshotRequest {
       }];
   // Seconds to wait when polling for updates.
   int32 period_seconds = 5;
+  // A range either side of batches_processed to include near-misses.
+  int32 batches_margin = 6;
```

Comment:
**nit:** What do you think about putting `batches_margin` right after `batches_processed`?

Reply:
you cant change the order or it becomes a breaking change to grpc. which is probably fine, but i imagine that's why it is here.

Reply:
Yeah typically when adding new fields to a protobuf you always go in increasing order, and changing what an existing number field means is an incompatibility. Of course, having seen our JavaScript bindings and the fact that they rely exclusively on the order of fields when making calls, and with the fact that this API is marked as Internal, I thought it would be best. I'll look into how to force an incompatible change through...

Reply:
Oh it's just `make -C proto gen-buf-image`